examples/mobile/Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [package]
 name = "bevy_mobile_example"
-version = "0.1.0"
```

Comment:
this one is needed by cargo-apk

Reply:
Fixed in 319db5130192e0eda5be483e32be09caf7617e6d :)